cid-redirects.json

```diff
@@ -2641,8 +2641,83 @@
+  "/docs/dashboards-new/set-custom-time-ranges": "/docs/dashboards/set-custom-time-ranges",
+  "/docs/dashboards-new/scheduled-report": "/docs/dashboards/scheduled-report",
+  "/docs/dashboards-new/export-dashboard-new": "/docs/dashboards/export-dashboard-new",
+  "/docs/dashboards-new/link-dashboards": "/docs/dashboards/link-dashboards",
+  "/docs/dashboards-new/set-data-access-level": "/docs/dashboards/set-data-access-level",
+  "/docs/dashboards-new/share-dashboard-new": "/docs/dashboards/share-dashboard-new",
+  "/docs/dashboards-new/share-dashboard-outside-org": "/docs/dashboards/share-dashboard-outside",
+  "/docs/dashboards-new/filter-template-variables": "/docs/dashboards/filter-template-variables",
+  "/docs/dashboards-new/advanced": "/docs/dashboards/advanced",
+  "/docs/dashboards-new/faq": "/docs/dashboards/faq",
+  "/docs/dashboards-new/dashboards-migration": "/docs/dashboards/dashboards-migration",
+  "/docs/dashboards-new/dashboards-migration/steps-to-migrate": "/docs/dashboards/dashboards-migration/steps-to-migrate",
+  "/docs/dashboards-new/dashboards-migration/changes-after-migration": "/docs/dashboards/dashboards-migration/changes-after-migration",
+  "/docs/dashboards-new/panels": "/docs/dashboards/panels",
+  "/docs/dashboards-new/panels/modify-chart": "/docs/dashboards/panels/modify-chart",
+  "/docs/dashboards-new/panels/area-charts": "/docs/dashboards/panels/area-charts",
+  "/docs/dashboards-new/panels/bar-charts": "/docs/dashboards/panels/bar-charts",
+  "/docs/dashboards-new/panels/box-plot-charts": "/docs/dashboards/panels/box-plot-charts",
+  "/docs/dashboards-new/panels/bubble-charts": "/docs/dashboards/panels/bubble-charts",
+  "/docs/dashboards-new/panels/column-charts": "/docs/dashboards/panels/column-charts",
+  "/docs/dashboards-new/panels/combo-charts": "/docs/dashboards/panels/combo-charts",
+  "/docs/dashboards-new/panels/line-charts": "/docs/dashboards/panels/line-charts",
+  "/docs/dashboards-new/panels/map-charts": "/docs/dashboards/panels/map-charts",
+  "/docs/dashboards-new/panels/markdown-syntax": "/docs/dashboards/panels/markdown-syntax",
+  "/docs/dashboards-new/panels/pie-charts": "/docs/dashboards/panels/pie-charts",
+  "/docs/dashboards-new/panels/sankey-charts": "/docs/dashboards/panels/sankey-charts",
+  "/docs/dashboards-new/panels/scatter-charts": "/docs/dashboards/panels/scatter-charts",
+  "/docs/dashboards-new/panels/single-value-charts": "/docs/dashboards/panels/single-value-charts",
+  "/docs/dashboards-new/panels/table-charts": "/docs/dashboards/panels/table-charts",
+  "/docs/dashboards": "/docs/dashboards-classic",
+  "/docs/dashboards/about": "/docs/dashboards-classic/about",
+  "/docs/dashboards/share-dashboard-inside-org": "/docs/dashboards-classic/share-dashboard-inside-org",
+  "/docs/dashboards/share-dashboard-outside-org": "/docs/dashboards-classic/share-dashboard-outside-org",
```

Comment:
@JV0812 does this line need to be fixed?

Reply:
@kimsauce We don't want to change anything here.